### PR TITLE
Update bug report issue template

### PR DESCRIPTION
**Pull request by erimatnor**
Convert the bug report issue template into a form that the user has to
fill out. The form allows making some fields required.

Also add the "bug" and "triage" labels to all bug reports. The
"triage" label can be used to find bugs that haven't been triaged,
assuming the label is removed as part of the triaging process.

### CODE REVIEW

**Comment by erimatnor** [comment]
It's possible to try this form out in my private repo: https://github.com/erimatnor/timescaledb/issues/new/choose

**Review by mkindahl — APPROVED** [review]
Will you update the bug procedure documentation to reflect the usage of the "triage" label?

**Review by NunoFilipeSantos — APPROVED** [review]

**Comment by erimatnor** [comment]
> Will you update the bug procedure documentation to reflect the usage of the "triage" label?

Good point. I will do that.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+  - type: dropdown
+    id: type
+    attributes:
+      label: What type of bug is this?
+      multiple: true
+      options:
+        - Crash
+        - Performance issue
+        - Incorrect query result
```
Comment (mkindahl) [inline .github/ISSUE_TEMPLATE/bug_report.yml:20]:
```suggestion
        - Incorrect result
```

**Review by mkindahl — APPROVED** [review]

**Pull request merged**